### DISCUSSION TITLE
Saludos a la comunidad!

### DISCUSSION THREAD

**Discussion by marcianotech** [marcianotech/prompts#1] [Announcements]
¡Hola a todos y bienvenidos al foro de discusión de GitHub de Oficial Marciano Tech! 🚀

Mi nombre es Marciano, y estoy emocionado de presentarles este espacio donde nuestra comunidad puede reunirse, colaborar e intercambiar ideas. Nuestro objetivo principal es desarrollar y mejorar proyectos relacionados con el trading, aplicaciones de inteligencia artificial y ChatGPT.

Nos encantaría que todos ustedes, los entusiastas y expertos en estas áreas, participen activamente en este foro. No importa si eres un principiante o un profesional, valoramos todas las perspectivas y estamos ansiosos por aprender de sus conocimientos y experiencias.

Aquí hay algunas formas en que pueden contribuir en el foro:

Comentar y discutir: Compartan sus opiniones, ideas y conocimientos en las conversaciones existentes. Estamos convencidos de que su aporte enriquecerá las discusiones y fomentará un ambiente de aprendizaje colaborativo.

Sugerencias de proyectos: Si tienen una idea para un nuevo proyecto o una mejora en uno de nuestros proyectos existentes, por favor compártanla con la comunidad. Estamos abiertos a explorar nuevas oportunidades y siempre buscamos formas de mejorar.

Ayuda y soporte: Si tienen preguntas o encuentran algún problema con nuestros proyectos o aplicaciones, no duden en pedir ayuda. Estamos aquí para apoyarnos mutuamente y crecer como comunidad.
Compartir recursos: Si encuentran algún recurso útil, como tutoriales, artículos o herramientas, no duden en compartirlos con el grupo. Estamos aquí para aprender juntos y mantenernos actualizados sobre las últimas tendencias e innovaciones.

Esperamos que este foro se convierta en un lugar de intercambio de conocimientos, crecimiento y, como dicen en el mundo del trading, en un espacio donde juntos "compramos en las caídas y vendemos en las subidas". Agradecemos sus contribuciones y nos entusiasma ver cómo nuestra comunidad se une para crear proyectos asombrosos e innovadores.

¡Gracias por unirse y estamos deseosos de ver sus aportaciones!

Saludos cordiales,

Marciano Finanzas

**Comment by B14cool** [comment]
Esto llega justo cuando estoy hypeado creando aplicaciones webs y buscando la manera de utilizar el API de ChatGPT o GPT 3!

**Reply by CDMM1998** [reply]
**trabajas de manera individual?
yo quiero montar software y venderlos**

**Comment by wilsontavarez809** [comment]
Buenas noche desde RD, estoy interesado en el trading con chat gpt

**Reply by navaja** [reply]
Este promp lo voy a probar en el system en la api de chatgpt a ver cómo se comporta y si sale de su papel de profesor o se desvía de tema ya que el modelo gpt 3.5 turbo no acepta entrenamiento fino. Como los modelos anteriores como davinci por ejemplo. 
Estaría bien compartir distintos configuración en el system para dar distintas personalidades a las aplicaciones.

**Comment by SpartGaming** [comment]
Buenas a todos , he pasado trabajo para encontrar el enlace del ChatGPT y usarlo
